package/ci/emscripten.sh

```diff
@@ -97,6 +97,8 @@ cmake .. \
     -DWITH_ICOIMPORTER=ON \
     -DWITH_JPEGIMAGECONVERTER=OFF \
     -DWITH_JPEGIMPORTER=OFF \
+    -DWITH_KTXIMAGECONVERTER=OFF \
```

Comment:
Why is it `OFF` here? :eyes:

Reply:
Should be enough to list all files it needs in the `FILES` section of `corrade_add_test()` similarly as you do for the importer test, e.g.

```cmake
corrade_add_test(KtxImageConverterTest KtxImageConverterTest.cpp
    LIBRARIES Magnum::Trade
    FILES
        ../../KtxImporter/Test/1d.ktx2
        ...)
```

That'll make Emscripten embed the files in the virtual filesystem which then the test can directly read. Though not sure if the relative paths work, if not then `${PROJECT_SOURCE_DIR}/src/MagnumPlugins/KtxImporter/`Test` instead.